### DISCUSSION TITLE
Video should stop playing when scrolled of screen

### DISCUSSION THREAD

**Discussion by MehBlorg** [robertodoering/harpy#772] [Ideas]
Play a video in a tweet, then scroll it off the timeline so it's not visible. The video keeps playing for a bit. It should stop immediately when scrolled out of view.

**Comment by scottiefalkon** [comment]
It does stop for me. Has for a long time.

**Comment by dnestico** [comment]
It does already I think

**Comment by MehBlorg** [comment]
"... keeps playing for a bit. It should stop immediately."

**Reply by robertodoering** [reply]
It will stop as soon as the Tweet card is scrolled off-screen.
Pausing the video on the exact pixel when it's scrolled off-screen is possible, but comes with a slight performance impact that I'd rather avoid seeing as the current behavior is completely fine.

**Reply by MehBlorg** [reply]
K. No worries.

Thanks for the consideration.
